src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,55 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    public static final int DATE_INDEX = 0;
+    public static final int NAME_INDEX = 1;
+    public static final int HOURS_INDEX = 2;
+    public static final int SALARY_RATE_INDEX = 3;
+    public static final int TOTAL_PARAMETERS = 4;
```

Comment:
do you use this variable somewhere?

Reply:
yes, I use them when I parse the String to refer to, e.g. name index of a parsed string. I thought it would be better then just using 1,2 etc, as it is clear from the name what I refer to

Reply:
Oh, I no longer use TOTAL_PARAMETERS, you are completely right, my excuses